README.md

```diff
@@ -39,6 +46,11 @@ conan remote add conancenter https://center.conan.io
 conan remote add ess-dmsc https://artifactory.esss.lu.se/artifactory/api/conan/ecdc-conan-release
 ```
 
+If on Ubuntu 22, install:
+```bash
+sudo apt install libqt5svg5-dev
```

Comment:
I might even simplify this `Building` section with simply a pointer to the circleci script to see what packages are installed. What bulids on CI is the closest thing to a guarantee of functionality. Everything else is hard to state with any certainty and/or maintain.